core/src/token_info/hardcoded.rs

```diff
@@ -0,0 +1,115 @@
+//! This module contains fallback token data that should be used by the price
```

Comment:
This file has been moved 1:1 from `core/src/price_estimation/data.rs` (TokenBaseInfo has been moved up into `core/src/token_info.rs`).